app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -364,8 +367,10 @@ public void onClick(DialogInterface dialog, int which) {
                 // Place a call
                 EditText contact = (EditText) ((AlertDialog) dialog).findViewById(R.id.contact);
                 params.put("to", contact.getText().toString());
+
                 ConnectOptions connectOptions = new ConnectOptions.Builder(accessToken)
                         .params(params)
+                        .preferAudioCodecs(Arrays.asList(new OpusCodec(), new PcmuCodec()))
```

Comment:
The API is not something I expect developers to use by default. We should document it in the PR but not make it a default part of the quickstart.

Reply:
ok.